src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/vuex/RunReportPageTests.kt

```diff
@@ -52,12 +66,29 @@ class RunReportPageTests : SeleniumTest()
     }
 
     @Test
-    fun `can view git branches`()
+    fun `can change git branch to non-master and see different git commits`()
```

Comment:
I also wanted to add a test that the selected commit would be maintained after switching tabs, but each branch has only one associated commit currently, so cannot test.